### DISCUSSION TITLE
How do i link my sig-release issues into my group / team repositories?

### DISCUSSION THREAD

**Discussion by FaGru3n** [eclipse-tractusx/sig-release#227] [Q&A] [answered]
Hi all, 

at the moment i got many requests for "How to link my planned Features / Bugs / Release Criteria" with the dedicated working issues on component repositories? 

Basic Idea:
Issues on sig-release for all Features / Bugs (when don´t know where to adress)  and Release Criteria Issues for Release Management.

So how do i link / menition my team issue to the Feature or Bug Issues on sig-release repository?

**Comment by stephanbcbauer** [comment]
@FaGru3n here is a good example https://github.com/eclipse-tractusx/sig-release/issues/212. This is a feature (created on sig-release) which links on issues, which are created within a specific repository.

Linking between issues across different repositories works really simple :) for example linking an issue from eclipse-tractusx/item-relationship-service#187

See documentation [here](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/autolinked-references-and-urls#issues-and-pull-requests)

**Comment by FaGru3n** [comment, marked as answer]
Hi, 

just to show how you can link your issues on sig-release repository to the dedicated working issue in your team repositories.

There are more options to link them in your team repositories, in general you can reference any issue based on the patterns that are provided by GitHub itself. [GitHub Docs](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/autolinked-references-and-urls#issues-and-pull-requests)

In our case you can like directly with in your team-reapository by only typing `#` and then the related issuenumber.

1. Example to link directly in the same repository:
`#68` will look like https://github.com/eclipse-tractusx/sig-release/issues/68

From team / component repo perspective it could be linked to your Feature with `GitHubOrg/repository#issuenumber` to reference the feature or the related issue on sig-release repository.

2. Example to link your team repo issue to the sig-release issue:
`eclipse-tractusx/sig-release#68` will look like https://github.com/eclipse-tractusx/sig-release/issues/68

3. Example to do the same as in example 2 shown with full urls
`https://github.com/eclipse-tractusx/sig-release/issues/68` will look like https://github.com/eclipse-tractusx/sig-release/issues/68

In Case you tranfered this issue from sig-release repo to your personal team repository be sure that the working issue is not linked in the "Release Planning" Board 

I guess now you can also adapt this in your tasklists in your feature issue in sig-release 

- [ ] #68 
- [ ] #212 

Some Examples were already created to demonstrate how this works see #212  #68
